src/resources/payment_method_ext.rs

```diff
@@ -22,4 +22,14 @@ impl PaymentMethod {
     ) -> Response<PaymentMethod> {
         client.post_form(&format!("/payment_methods/{}/attach", payment_method_id), params)
     }
+
+    /// Detach a payment method from its customer
+    ///
+    /// For more details see [https://stripe.com/docs/api/payment_methods/detach](https://stripe.com/docs/api/payment_methods/detach).
+    pub fn detach(
+        client: &Client,
+        payment_method_id: &PaymentMethodId,
+    ) -> Response<PaymentMethod> {
+        client.post_form(&format!("/payment_methods/{}/detach", payment_method_id), ())
```

Comment:
As there is no `form` best to use the `post` method. This seems to work ok for me.

```suggestion
        client.post(&format!("/payment_methods/{}/detach", payment_method_id))
```